src/components/PublishArticle.jsx

```diff
@@ -4,42 +4,59 @@ import UpdateArticle from "./UpdateArticle";
 import createHeaders from "../modules/headers";
 import fetchArticle from "../modules/fetchArticle";
 import { useDispatch } from "react-redux";
+import { Container } from 'semantic-ui-react'
 
 const PublishArticle = (props) => {
   const dispatch = useDispatch();
   const [message, setMessage] = useState("");
+  const [loading, setLoading] = useState(false);
 
   useEffect(() => {
     fetchArticle(dispatch, setMessage, props.match.params.id);
   }, []);
 
   const onSubmitHandler = async (e) => {
-    try {
-      const response = await axios.put(
-        `/admin/articles/${props.match.params.id}`,
-        {
-          activity: "PUBLISH",
-          premium: e.target.premium.value,
-          category: document
-            .getElementById("category")
-            .firstElementChild.innerText.toLowerCase(),
-        },
-        {
-          headers: createHeaders(),
-        }
-      );
-      setMessage(response.data.message);
-    } catch (error) {
-      setMessage(error.response.data.message);
+    const isLocal = e.target.local.checked;
+    const isInternational = e.target.international.checked;
+
+    if (isLocal === false && isInternational === false) {
+      return setMessage("Please select either local or international");
```

Comment:
Couldn't this be done by international default checked and disabled, until local is checked, then it is enabled?